src/components/dashboard/dashboard.service.ts

```diff
@@ -1124,6 +1124,9 @@ export class DashboardService implements OnDestroy {
             return true;
         }
 
+        // emit information about the layout
+        this.layout$.next(this.getLayoutData());
```

Comment:
We should also emit this in the `removeWidget` function as well as it looks like the widget is getting left behind in the layout data after a widget has been removed.